domain-ee/ee-ep-merge-app/src/python_src/schema/merge_job.py

```diff
@@ -10,22 +10,22 @@
 
 class JobState(str, Enum):
     PENDING = 'PENDING'
-    RUNNING_GET_PENDING_CLAIM = 'RUNNING_GET_PENDING_CLAIM'
-    RUNNING_GET_PENDING_CLAIM_CONTENTIONS = 'RUNNING_GET_PENDING_CLAIM_CONTENTIONS'
-    RUNNING_GET_EP400_CLAIM_CONTENTIONS = 'RUNNING_GET_EP400_CLAIM_CONTENTIONS'
-    RUNNING_SET_TEMP_STATION_OF_JURISDICTION = 'RUNNING_SET_TEMP_STATION_OF_JURISDICTION'
-    RUNNING_MERGE_CONTENTIONS = 'RUNNING_MERGE_CONTENTIONS'
-    RUNNING_MOVE_CONTENTIONS_TO_PENDING_CLAIM = 'RUNNING_MOVE_CONTENTIONS_TO_PENDING_CLAIM'
-    RUNNING_CANCEL_EP400_CLAIM = 'RUNNING_CANCEL_EP400_CLAIM'
-    RUNNING_ADD_CLAIM_NOTE_TO_EP400 = 'RUNNING_ADD_CLAIM_NOTE_TO_EP400'
+    GET_PENDING_CLAIM = 'GET_PENDING_CLAIM'
+    GET_PENDING_CLAIM_CONTENTIONS = 'GET_PENDING_CLAIM_CONTENTIONS'
+    GET_EP400_CLAIM_CONTENTIONS = 'GET_EP400_CLAIM_CONTENTIONS'
+    SET_TEMP_STATION_OF_JURISDICTION = 'SET_TEMP_STATION_OF_JURISDICTION'
+    MERGE_CONTENTIONS = 'MERGE_CONTENTIONS'
+    MOVE_CONTENTIONS_TO_PENDING_CLAIM = 'MOVE_CONTENTIONS_TO_PENDING_CLAIM'
+    CANCEL_EP400_CLAIM = 'CANCEL_EP400_CLAIM'
+    ADD_CLAIM_NOTE_TO_EP400 = 'ADD_CLAIM_NOTE_TO_EP400'
     COMPLETED_SUCCESS = 'COMPLETED_SUCCESS'
 
-    RUNNING_GET_PENDING_CLAIM_FAILED_REMOVE_SPECIAL_ISSUE = 'RUNNING_GET_PENDING_CLAIM_FAILED_REMOVE_SPECIAL_ISSUE'
-    RUNNING_GET_PENDING_CLAIM_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE = 'RUNNING_GET_PENDING_CLAIM_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE'
-    RUNNING_SET_TEMP_STATION_OF_JURISDICTION_FAILED_REMOVE_SPECIAL_ISSUE = 'RUNNING_SET_TEMP_STATION_OF_JURISDICTION_FAILED_REMOVE_SPECIAL_ISSUE'
-    RUNNING_MOVE_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE = 'RUNNING_MOVE_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE'
-    RUNNING_MOVE_CONTENTIONS_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION = 'RUNNING_MOVE_CONTENTIONS_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION'
-    RUNNING_CANCEL_CLAIM_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION = 'RUNNING_CANCEL_CLAIM_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION'
+    GET_PENDING_CLAIM_FAILED_REMOVE_SPECIAL_ISSUE = 'GET_PENDING_CLAIM_FAILED_REMOVE_SPECIAL_ISSUE'
+    GET_PENDING_CLAIM_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE = 'GET_PENDING_CLAIM_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE'
+    SET_TEMP_STATION_OF_JURISDICTION_FAILED_REMOVE_SPECIAL_ISSUE = 'SET_TEMP_STATION_OF_JURISDICTION_FAILED_REMOVE_SPECIAL_ISSUE'
+    MOVE_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE = 'MOVE_CONTENTIONS_FAILED_REMOVE_SPECIAL_ISSUE'
+    MOVE_CONTENTIONS_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION = 'MOVE_CONTENTIONS_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION'
+    CANCEL_CLAIM_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION = 'CANCEL_CLAIM_FAILED_REVERT_TEMP_STATION_OF_JURISDICTION'
```

Comment:
This is so Java-ish but, then again, DRYing it up won't actually reduce the LoC count.

Reply:
Sorry can't help myself! 😆